src/main/reactjs/src/components/layouts/Footer.tsx

```diff
@@ -9,12 +9,18 @@ import Logo from 'public/assets/svg/logo.svg';
 import FooterWave from 'public/assets/svg/footer_wave.svg';
 import { useAppTranslation } from 'configs/i18n';
 
-const Footer: FC = () => {
+interface FooterProps {
+  showWave: boolean;
+}
+
+const Footer: FC<FooterProps> = ({ showWave }: FooterProps) => {
   const { t } = useAppTranslation({ keyPrefix: 'akt.component.footer' });
 
   return (
     <footer>
-      <Svg className="footer__wave" src={FooterWave} alt={t('logo.alt')} />
+      {showWave && (
+        <Svg className="footer__wave" src={FooterWave} alt={t('logo.alt')} />
+      )}
```

Comment:
Tämä voisi toteuttaa sitten kun meillä on tieto kirjautumisesta redux tilassa. Hyvin todennäköisesti piilotetaan footeri kokonaan, koska siellä ei ole hyödyllistä tietoa virkailijoille.

Reply:
Joo, varmaan tosiaan näin. Toisaalta jos ei haittaa suuresti, niin pitäisin tämän tällaisena kunnes design tarkentuu.

Reply:
juu, tämä on ok vaihdetaan myöhemmin.